### ISSUE TITLE
Accessibility swarm on the Live Label component

### ISSUE THREAD

**Issue by DenisHdz** [BBC-archive/psammead#3230]
**Is your feature request related to a problem? Please describe.**
Carry out a 2 person developer A11Y swarm for the `LiveLabel` component using the `StoryPromo` and `Bulletin` components https://github.com/bbc/psammead/pull/3264
 
**Describe the solution you'd like**
Go through this [checklist](https://bbc-news.github.io/accessibility-news-and-you/accessibility-news-and-testers) and fix any issue we might face.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by DenisHdz** [comment]
❓ = Not done
✅ = Done, pass
❌ = Done, fail
👀 = Review with UX

## Checklist - Manual ##

  - ### Design ✅

  - ### Color contrast ✅

<img width="855" alt="color-contrast" src="https://user-images.githubusercontent.com/46446236/77330951-f3ca0580-6d17-11ea-8b44-fb44acf945e4.png">

  - ### Color blindness ✅

  - ### High Contrast mode PC ✅

  - ### Color preferences changed in Firefox ✅
    
  - ### Text resized to 200% ✅

  - ### Page zoomed to 200% ✅

  - ### Heading order ✅ 

  - ### Icons ✅ 
   
  - ### Keyboard navigation: tab order, focus css, trace tab path ✅ 
![image](https://user-images.githubusercontent.com/30599794/77331380-8e2a4900-6d18-11ea-87b0-95456a2e0da8.png)

  - ### aXe ✅ 
![image](https://user-images.githubusercontent.com/30599794/77331129-35f34700-6d18-11ea-8676-05a2275be440.png)


  - ### No CSS ✅ 
![image](https://user-images.githubusercontent.com/30599794/77331043-165c1e80-6d18-11ea-9b7f-0233f77eadf8.png)

  - ### No Javascript ❓ 

  - ### bbc-a11y ✅ 
![image](https://user-images.githubusercontent.com/30599794/77331083-2673fe00-6d18-11ea-859c-612e60a21666.png)


## Checklist - Assistive Technology ##

Acceptance criteria:
```
Given I use a screen reader
When I navigate to a live promo
And I am navigating via headings and links
Then the live label should be announced as part of the heading/link
And the live label should be announced first

Given I use a screen reader (LTR languages)
When viewing a live promo
The ‘Live’ label should be read in English, with an English accent
And it should be pronounced as ‘live’ as in ‘live event’ rather than ‘live’ as in ‘I live here’ 
```

 - ###  VoiceOver OS (Mac) (Screen Reader)

   > With Safari (Latest Version) on Mac OS (Latest Version)  

 - ### VoiceOver iOS (iPhone/iPad) (Screen Reader) 

    > Latest Version with Safari (Latest Version) on iOS (Latest Version)

 - ### TalkBack  (Screen Reader) ✅
 
   > Latest Version with Chrome (Latest Version) on Android (Latest Version)

- ### ZoomText (Screen Magnifier with Screen Reader capabilities) ✅

  > Latest Version with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

    Live reads as "Live liv <headline text>" -> We can ignore this

- ### Dragon Naturallyspeaking  (Speech Recognition) ✅

   > Version 13 with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

- ### JAWS  (Screen Reader)  ✅

   > Version 17 with Internet Explorer 11 on Windows (XP/Vista/7/8/10) 

- ### Read&Write (Reading Solution) ✅

   > Latest Version with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

- ### NVDA  (Screen Reader) ✅

   > Latest Version with Firefox (Latest Version) on Windows (XP/Vista/7/8/10)

    Live reads as "I live here" rather than "Live event" when navigating via Heading and links, 
    read as expected with the arrow key -> We can ignore this

**Comment by thekp** [comment]
Testing: 
VoiceOver OS (Mac) (Screen Reader) ✅ 
> With Safari (Latest Version) on Mac OS (Latest Version)

VoiceOver iOS (iPhone/iPad) (Screen Reader) (in-progress) ✅ 
> Latest Version with Safari (Latest Version) on iOS (Latest Version)

I checked the link and heading for: LIVE Bullentins and LIVE storypromos for RTL and LTR.

**Comment by DenisHdz** [comment]
This can be closed after a successfull a11y swarm.
